samza-kafka/src/test/scala/org/apache/samza/checkpoint/kafka/TestKafkaCheckpointManager.scala

```diff
@@ -248,8 +305,10 @@ class TestKafkaCheckpointManager extends KafkaServerTestHarness {
     socketTimeout = 30000,
     bufferSize = 64 * 1024,
     fetchSize = 300 * 1024,
+    getSystemConsumer = systemConsumerF,
+    getSystemAdmin = systemAdminF,
     metadataStore = metadataStore,
-    connectProducer = () => new KafkaProducer(producerConfig.getProducerProperties),
+    getSystemProducer = systemProducerF,
```

Comment:
s/systemProducerF/systemProducerFn

Like wise, replace for all occurences of consumer and admin

Reply:
done.